### DISCUSSION TITLE
copilot not activated

### DISCUSSION THREAD

**Discussion by MiracleShadow** [community/community#8814] [Copilot Conversations] [answered]
After install and enable, it did not work as expected.

```
[INFO] [default] [2021-12-14T03:05:53.532Z] Experimentation service initialized
[INFO] [default] [2021-12-14T03:05:53.536Z] Feature for copilotaa: false
[INFO] [default] [2021-12-14T03:05:53.536Z] Feature for copilotincludeprompt: none
```

**Comment by davecheney** [comment, marked as answer]
This could be network related — are you behind a firewall or other technology that may be intercepting your HTTPS traffic?

**Reply by MiracleShadow** [reply]
Thank you for your answer, yes, my environment uses proxy, but I have no problem accessing the website and downloading.

**Reply by davecheney** [reply]
Sadly copilot does not support proxies at the moment.

**Reply by MiracleShadow** [reply]
Ok, thank you. I hope we can get support as soon as possible. Do you have any relevant plans at present?😁

**Reply by MiracleShadow** [reply]
oh,I know what happend
```
workbench.desktop.main.js:355419 Activating extension 'GitHub.copilot' failed: Cannot read property 'message' of undefined.
```
Looks like someone had the same problem: https://github.com/github/feedback/discussions/8587
